calplus/tests/unit/v1/compute/drivers/test_openstack_driver.py

```diff
@@ -0,0 +1,399 @@
+""" OpenstackDriver for Compute
+    based on BaseDriver fror Compute Resource
```

Comment:
https://github.com/cloudcomputinghust/CAL/commit/1803c6cebf43e2fb38fd5ed5a13c87d28bd269e5#diff-e4ec7c2336a8631cc0f5c4f145293979R2

I just fixed it by this commit.